pkg/collector/corechecks/cluster/orchestrator/processors/k8s/base.go

```diff
@@ -0,0 +1,18 @@
+package k8s
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/processors"
+)
+
+type BaseHandler struct{}
```

Comment:
I only added the methods to the baseHandler which are seldom (`ScrubBeforeMarshalling`, `BeforeCacheCheck`) or not at all (`BeforeMarshalling`) overwritten